dont-worry-be-happy/dwbh-api
# Add security to get group

![](https://media2.giphy.com/media/m9rMVyKactNK/giphy.gif?cid=3640f6095c9c9caf47315a7a41ca4fb4)

# HOW TO TEST
Try to call to getGroup with an id of a group in which the user doesn't belongs. It should fail
